React/Fabric/Mounting/ComponentViews/ActivityIndicator/RCTActivityIndicatorViewComponentView.mm

```diff
@@ -28,7 +29,7 @@ static UIActivityIndicatorViewStyle convertActivityIndicatorViewStyle(const Acti
 }
 
 @implementation RCTActivityIndicatorViewComponentView {
-  UIActivityIndicatorView *_activityIndicatorView;
+  RCTActivityIndicatorView *_activityIndicatorView;
```

Comment:
Should this be something like RCTUIActivityIndicatorView?

Reply:
We are only re-using the current shims in this re-naming pass.

Reply:
Mentioned elsewhere in #1539 , this would be a change to iOS / logic since `UIActivityIndicatorView` != `UIActivityIndicatorView`. I'm not sure we're ok with that at this moment, so I'm gonna go and discuss with my team about it.

Reply:
I'll make a similar re-name like we did for `RCTUISlider` in #1525